server/game/cards/06-TLS/MichaelTheBadgerDodge.js

```diff
@@ -0,0 +1,73 @@
+const DudeCard = require('../../dudecard.js');
+const GameActions = require('../../GameActions/index.js');
+
+/** @typedef {import('../../AbilityDsl')} AbilityDsl */
+
+class MichaelTheBadgerDodge extends DudeCard {
+    /** @param {AbilityDsl} ability */
+    setupCardAbilities(ability) {
+        this.persistentEffect({
+            targetController: 'current',
+            condition: () => this.hasAttachmentWithKeywords(['melee', 'weapon']), 
+            match: this,
+            effect: [
+                ability.effects.setAsStud()
+            ]
+        });
+        this.action({
+            title: 'Michael "The Badger" Dodge',
+            playType: ['shootout'],
+            cost: [
+                ability.costs.bootSelf()
+            ],
+            target: {
+                activePromptTitle: 'Choose an opposing Dude',
+                choosingPlayer: 'current',
+                cardCondition: { 
+                    inOpposingPosse: true, 
+                    controller: 'opponent',
+                    condition: card => !card.booted
+                },
+                cardType: ['dude']
+            },
+            message: context => this.game.addMessage('{0} uses {1} to boot an opposing dude', context.player, this),
```

Comment:
The PR looks good, just one last change:
```suggestion
            message: context => this.game.addMessage('{0} uses {1} to boot {2}', context.player, this, context.target),
```